app/experimenter/static/visualization/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@experimenter/visualization",
+  "license": "MPL-2.0",
+  "version": "0.0.0",
+  "scripts": {
+    "build": "npm run clean; rollup --config",
+    "clean": "rimraf ../assets/visualization",
+    "watch": "rollup --config --watch",
+    "format": "prettier --plugin-search-dir=. --write .",
+    "format:check": "prettier --plugin-search-dir=. --check .",
+    "lint": "eslint --ext .js,.svelte ."
+  },
+  "devDependencies": {
+    "@babel/core": "^7.9.0",
+    "@babel/plugin-transform-runtime": "^7.9.0",
+    "@babel/preset-env": "^7.9.0",
+    "@graph-paper/box": "0.0.0-alpha.12",
+    "@graph-paper/button": "0.0.0-alpha.12",
+    "@graph-paper/core": "0.0.0-alpha.12",
+    "@graph-paper/datagraphic": "0.0.0-alpha.12",
+    "@graph-paper/elements": "0.0.0-alpha.12",
+    "@graph-paper/guides": "0.0.0-alpha.12",
+    "@graph-paper/icons": "0.0.0-alpha.12",
+    "@graph-paper/stack": "0.0.0-alpha.12",
+    "@graph-paper/tiles": "0.0.0-alpha.12",
+    "@rollup/plugin-commonjs": "^11.0.0",
+    "@rollup/plugin-node-resolve": "^7.0.0",
+    "@rollup/plugin-typescript": "^5.0.2",
+    "@tsconfig/svelte": "^1.0.10",
+    "babel-loader": "^8.1.0",
+    "d3-dsv": "^1.2.0",
+    "d3-format": "^1.4.4",
+    "d3-scale-chromatic": "^1.5.0",
+    "d3-time-format": "^2.2.3",
+    "eslint": "^7.0.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-jest": "^23.13.1",
+    "eslint-plugin-prettier": "^3.1.3",
+    "eslint-plugin-svelte3": "^2.7.3",
+    "npm-run-all": "^4.1.5",
+    "page": "^1.11.5",
+    "prettier": "^2.0.5",
+    "prettier-plugin-svelte": "^1.1.0",
+    "prismjs": "^1.20.0",
+    "rimraf": "3.0.2",
+    "rollup": "^2.14.0",
+    "rollup-plugin-copy": "^3.3.0",
+    "rollup-plugin-livereload": "^1.0.0",
+    "rollup-plugin-postcss": "^2.5.0",
+    "rollup-plugin-svelte": "^5.0.3",
+    "rollup-plugin-terser": "^5.1.2",
+    "svelte": "^3.20.1",
+    "svelte-check": "^1.0.28",
+    "svelte-preprocess": "^4.1.2",
+    "tsconfig-paths": "^3.9.0",
+    "tslib": "^2.0.1",
+    "typescript": "^4.0.2"
+  },
+  "dependencies": {
+    "@google-cloud/bigquery": "^4.7.0",
+    "express": "^4.17.1",
+    "sirv-cli": "^0.4.4"
+  }
+}
```

Comment:
We might need to tweak the .github/dependabot.yml file to make sure dependabot depends these deps up to date.